source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
> The problem with that IIFE approach is that the event listener is set even if Likely is used as a CommonJS module.

Yep. I get it now. Module exists only within bundling runtime and not after that in browser runtime.